botfront/imports/api/graphql/botResponses/resolvers/nlgResolver.js

```diff
@@ -0,0 +1,98 @@
+import { safeLoad } from 'js-yaml';
+import { sample } from 'lodash';
+import BotResponses from '../botResponses.model';
+
+const subText = (text, slots) => {
```

Comment:
May be a more explicit function name that describes what the function does, something like `interpolateSlots`